components/html-block/html-block.js

```diff
@@ -5,6 +5,11 @@ import { HtmlBlockMathRenderer } from '../../helpers/mathjax.js';
 import { requestInstance } from '../../mixins/provider-mixin.js';
 
 export const htmlBlockContentStyles = css`
+	.d2l-html-block-compact {
```

Comment:
I'd suggest making the `compact` property reflect and then using attribute selectors to apply this CSS instead of needing to add and trigger off a CSS class.

```css
:host([compact]) {
  font-size: 0.8rem;
  /* etc */
}
```

Reply:
So the reason this is in the style exports is because it's also going to need to be imported into the HTML editor's edit mode, as is done here: https://github.com/BrightspaceUI/htmleditor/blob/main/htmleditor-mixin.js#L546

So I was looking for a solution that didn't require making identical style changes in both locations (keeping it common to the export) while also allowing it to remain static CSS.

Reply:
If you think the duplication is preferred, then that's also fine. I have no strong opinions here. The HTML editor does already import its own styles as well; there's just this block that's common to both because we want some of our styling to be consistently reflected in both components.

Reply:
Gotcha, OK that makes sense! I think it's fine then... the only other option would be something like:

```css
:host([compact]), .d2l-html-block-compact {
  /* etc. */
}
```

So one rule for this component and one rule for others to use. Of course if the others just happened to have a `compact` property weird things could ensue! I'm good with whatever.

Reply:
Yeah, that's sort of interesting. It's a bit unfortunate that we have to deal with the editor iframe, otherwise I think it would be much easier to do this sort of stuff. 😞